matrix_benchmarking/download_lts.py

```diff
@@ -34,28 +35,34 @@ def main(opensearch_host: str = "",
     opensearch_index: the OpenSearch index where the LTS payloads are stored (Mandatory)
 
     results_dirname: The directory to place the downloaded results files.
-    filters: If provided, only download the experiments matching the filters. Eg: expe=expe1:expe2,something=true. (Optional.)
+    filters: If provided, only download the experiments matching the filters. Eg: {"image_name": "1.2"}. (Optional.)
+    max_records: Maximum number of records to retrieve from the OpenSearch instance. 10,000 is the largest number possible without paging (Optional.)
     """
     kwargs = dict(locals()) # capture the function arguments
 
-    optionals_flags = ["filters"]
-    safe_flags = ["filters", "results_dirname", "opensearch_index"]
+    optionals_flags = ["filters", "max_records"]
+    safe_flags = ["filters", "results_dirname", "opensearch_index", "max_records"]
 
-    cli_args.setup_env_and_kwargs(kwargs)
+    cli_args.update_env_with_env_files()
+    cli_args.update_kwargs_with_env(kwargs)
```

Comment:
Since the fields in the payload that we may be filter against could be numeric values or string values I wanted to ensure there was a distinction between the two in the filter provided in the cli argument. The format of the previous filter `field1=foo` didn't have that distinction, so I just omitted this parsing step so it would be automatically parsed by the CLI as a dictionary.